blog/2020-05-11-version-0.5.0-released.md

```diff
@@ -0,0 +1,86 @@
+---
+id: version-0.5.0-released
+title: Version 0.5.0 released
+author: Beanow
+author_url: https://github.com/Beanow
+author_image_url: https://avatars3.githubusercontent.com/u/497556?v=4&s=200
+---
+
+It's been months since the last versioned release, so there's a lot new to
+v0.5.0!
+
+You can find the release [on GitHub][GitHub release] or as a [Docker image].
+More details on how to use them can be found in the [installation guide].
+
+Since this is a large release, I'll only touch on several major additions here.
+
+### Discourse plugin
+
+Discourse forums can be analyzed with this release. And from using it so far, I
+think it's a really interesting plugin.
+
+Discourse forums by default require no authentication to be set up. Meaning you
+can load most Discourse forums out there and explore them with SourceCred, no
+approval required. The [plugin setup guide][discourse setup] walks through the
+commands to do this.
+
+Much thanks to the straightforward 💚 likes on Discourse, the out-of-the-box
+settings tend to be quite reasonable. Most of the tweaking involves balancing
+the Discourse weight relative to other platforms.
+
+### Initiatives
+
+[Initiatives] are added to SourceCred by a new plugin of the same name.
+Instructions are in it's [setup guide][initiatives setup].
+
+While being technical to use in it's current version, it adds a number of new
+capabilities. Such as manually adding contributions for which there's no plugin,
```

Comment:
"... new capabilities, such as manually..."
This keeps it form being a sentence fragment

Reply:
Hm! Didn't know about sentence fragments. Having a paragraph-sized sentence also didn't seem great to me, so in this case I wanted to make the fragment a proper sentence instead. What do you think of this?

> Example of capabilities I'd like to look into are adding
contributions for which there's no plugin, incentivizing work before it's
completed and grouping contributions together to value them more easily.